less/components/popups.less

```diff
@@ -69,12 +67,54 @@
     align-items: center;
     width: 100%;
     height: 80px;
-    min-height: 80px;
+    flex-shrink: 0;
```

Comment:
This is news to me. If we see bugs on whatever is the minimum version of IE we support I guess we can address it there, but I'd go with an ie-specific stylesheet rather than dropping flex-box, as it's miles better and it's *everywhere* in the editor.